src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/.pre-commit-config.yaml

```diff
@@ -1,28 +1,28 @@
 repos:
```

Comment:
We don't have a consolidated place where all of these hooks are defined well. For example, on Volitare, we have poetry-check, which I think is useful: https://github.com/ansys-internal/volitare/blob/13be88cfb1e6a9950fb934f6c3e0d71f1dee6425/.pre-commit-config.yaml#L38-L42

On other repos, like actions, we have stuff like vale and markdownlint, which I also think are valuable: https://github.com/ansys-internal/solution-applications-actions/blob/e5959b58edc4b0cbac6185def241489b01a4a9bc/.pre-commit-config.yaml#L18-L35